Changes

```diff
@@ -28,11 +28,11 @@ v2020.1 2020-11-06 (public release version)
  - This version of Zonemaster also consists of the following components.
    For each component, see its Changes file or Github release notes.
 
-   Zonemaster-LDNS     v2.1.0   https://github.com/zonemaster/zonemaster-ldns
-   Zonemaster-Engine   v4.0.0   https://github.com/zonemaster/zonemaster-engine
-   Zonemaster-CLI      v3.0.0   https://github.com/zonemaster/zonemaster-cli
-   Zonemaster-Backend  v6.0.0   https://github.com/zonemaster/zonemaster-backend
-   Zonemaster-GUI      v3.2.1   https://github.com/zonemaster/zonemaster-gui
+   Zonemaster-LDNS     v2.1.0   https://github.com/zonemaster/zonemaster-ldns/releases
+   Zonemaster-Engine   v4.0.1   https://github.com/zonemaster/zonemaster-engine/releases
+   Zonemaster-CLI      v3.0.1   https://github.com/zonemaster/zonemaster-cli/releases
+   Zonemaster-Backend  v6.0.1   https://github.com/zonemaster/zonemaster-backend/releases
+   Zonemaster-GUI      v3.2.1   https://github.com/zonemaster/zonemaster-gui/releases
```

Comment:
I'm not sure what the idea is here. But since we're changing things, have you considered linking directly to the actual release?

Reply:
That is an option. Let us look at that for next release.